cmd/etlprov/README.md

```diff
@@ -3,9 +3,18 @@
 ## Usage
 
 ```bash
-$ pedsnet-etlprov [-model <model>] [-version <version>] [-service <service>] <dir>
+$ pedsnet-etlprov [-model <model>] [-version <version>] [-truncate false] [-ignore <entities>] [-service <service>] [-ver] <dir>
```

Comment:
I think it should be `[-truncate=false]`. At least it doesn't work without the "=" right now.